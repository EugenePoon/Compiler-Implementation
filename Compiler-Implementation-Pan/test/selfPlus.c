int main()
{
  int a;
  int b;
  a=1;
  b=1;
  print a;
  a++;
  print a;
  ++a;
  print a;
  b--;
  print b;
  --b;
  print b;
}