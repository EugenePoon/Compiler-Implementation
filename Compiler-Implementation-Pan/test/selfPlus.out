24 19 0 5 25 15 1 15 1 13 0 1 12 15 -1 13 0 1 1 0 1 12 15 -1 13 11 22 15 -1 13 11 13 13 11 0 1 1 12 15 -2 13 11 22 15 -1 13 11 0 1 1 15 -1 13 13 11 0 1 1 12 15 -1 13 11 22 15 -1 13 0 1 1 11 13 0 1 1 13 0 1 1 11 0 1 2 12 15 -2 13 0 1 1 11 22 15 -1 13 0 1 1 11 0 1 2 15 -1 13 0 1 1 13 0 1 1 11 0 1 2 12 15 -1 13 0 1 1 11 22 21 2